README.md

```diff
@@ -8,3 +8,61 @@ It includes Javascript and native stack traces, heap statistics,
 platform information and resource usage etc. With the report enabled,
 reports are triggered on unhandled exceptions, fatal errors, signals
 and calls to a Javascript API.
+
+Usage:
+
+    npm install nodejs/nodereport
+
+    var nodereport = require('nodereport');
+
+By default, this will trigger a NodeReport to be written to the current
+directory on unhandled exceptions, fatal errors or (Linux/OSX only)
+SIGUSR2 signals. In addition, a NodeReport can be triggered from a 
+Javascript application via API. The filename of the NodeReport is
+returned. The default filename includes the date, time, PID and a
+sequence number. Alternatively a filename can be specified on the API call.
+
+    nodereport.triggerReport();
+
+    var filename = nodereport.triggerReport();
+
+    nodereport.triggerReport("myReportName");
+
+Content of the NodeReport in the initial implementation consists of a
+header section containing the event type, date, time, PID and Node version,
+sections containing Javascript and native stack traces, a section containing
+V8 heap information, a section containing libuv handle information and an OS
+platform information section showing CPU and memory usage and system limits.
+The following messages are issued to stderr when a NodeReport is triggered:
+
+    Writing Node.js error report to file: NodeReport.201605113.145311.26249.001.txt
+    Node.js error report completed
+
+Configuration is available via these environment variables and/or Javascript
+API calls:
+
+    export NODEREPORT_EVENTS=exception+fatalerror+signal
```

Comment:
that its possible to configure nodereport via env vars is great, that its necessary to do so is I think too much focus on policy instead of mechanism